stl/inc/yvals_core.h

```diff
@@ -346,7 +346,7 @@
 #define _MSVC_KNOWN_SEMANTICS
 #endif
 
-// Controls whether the STL uses "if constexpr" internally
+// Controls whether the STL uses "if constexpr" internally in sub-C++17 modes

```

Comment:
There's only one:
```suggestion
// Controls whether the STL uses "if constexpr" internally in C++14 mode
```

Reply:
We have enough compiler flags that there are trillions of C++14 modes! (Change accepted.)